notebooks/lightning_segmentation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p><code>add --upgrade --quiet</code>  to the install flags. <code>--upgrade </code> is quite nice with colab because the runtime does not need to be reloaded everytime the cell is executed. <code>--quiet</code>  is just a preference inside of notebooks but the output is much cleaner :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PhoenixDL/rising/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>